### PR TITLE
Improve README.md

### PR DESCRIPTION
**Pull request by Panquesito7**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
-# item_party
+# Item Party
 
-The probably worst economic Minetest mod.
+The probably worst economic Minetest mod.
```
Comment (SmallJoker) [inline README.md:3]: Not an improvement.

File: README.md [hunk 2]
```diff
@@ -25,9 +25,7 @@ huge node quantities.
 `item_drop` is a nice addition to this annoying mod. You can prevent players
 from doing something serious by feeding them useless items.
 
-
 ## Licensing
 
-Code: CC0/WTFPL
-
+Code: CC0/WTFPL
```
Comment (SmallJoker) [inline README.md]: The empty new line was intended to add a new line in the markdown rendered output. Please have a look at this file in a markdown editor.

Comment (Panquesito7) [reply README.md]: Changes reverted.

File: README.md (hunk 1 above)
Comment (Panquesito7) [reply README.md:3]: Without this "improvement" (in markdown format):

        The probably worst economic Minetest mod. Various common nodes have an unusual behaviour:

With this "improvement" (in markdown format):
```
The probably worst economic Minetest mod. 
Various common nodes have an unusual behaviour:
```

Which one do you choose?

Comment (SmallJoker) [reply README.md:3]: til `   ` results in a line continuation rather than a full new line. Never mind then, no need to change.

**Pull request merged**